server/src/main/java/io/crate/metadata/ColumnIdent.java

```diff
@@ -582,4 +582,14 @@ public void validForCreate() {
         path().forEach(x -> validateObjectKey(x));
     }
 
+    public List<String> extractPathFromParent(ColumnIdent parent) {
```

Comment:
```suggestion
    public List<String> getRelativePath(ColumnIdent parent) {
```

To match other methods in this class which use `get` prefixes.